sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubAuthenticationPolicy.cs

```diff
@@ -47,5 +54,16 @@ public override void OnSendingRequest(HttpMessage message)
 
             message.Request.Headers.SetValue(HttpHeader.Names.Authorization, headerValue);
         }
+
+        private sealed class KeyBytesCache
+        {
+            public KeyBytesCache(string key)
+            {
+                Key = key;
+                KeyBytes = Encoding.UTF8.GetBytes(key);
```

Comment:
```suggestion
                KeyBytes = string.IsNullOrEmpty(key) 
                    ? Array.Empty<byte>() 
                    : Encoding.UTF8.GetBytes(key);
```

Probably a tiny efficiency, but should we consider avoiding `GetBytes` when the string is empty?   It looks like there's no special case for it and it'll do some stack allocations that we can avoid.  _([ref](https://source.dot.net/#System.Private.CoreLib/UTF8Encoding.Sealed.cs,53))_    Maybe a factory method for an empty instance, instead?